spec/models/atmosphere/virtual_machine_spec.rb

```diff
@@ -323,6 +323,22 @@ def delete_with_success_when_exception(e)
     end
   end
 
+  context '::monitorable' do
+    let!(:vm) { create(:virtual_machine, ip: priv_ip) }
+    let!(:managable_vm) { create(:virtual_machine, managed_by_atmosphere: true,
+      ip: priv_ip_2) }
+    let!(:monitorable_vm) { create(:virtual_machine,
+      managed_by_atmosphere: true, monitoring_id: '1', ip: public_ip) }
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Expression at 331, 71 should be on its own line.